groups/sig-contributor-experience/OWNERS

```diff
@@ -0,0 +1,10 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
```

Comment:
```suggestion
reviewers:
```
I would happily merge this PR as-is if this wasn't present.  I do want to open up the pool but we're not there yet.

If the goal is to fast-track membership changes, I would suggest we test against a known list of groups that is under /groups/OWNERS approval only

If the goal is to allow sigs/wgs/whatever to create whatever @kubernetes.io groups they want, I would at a minimum require adherence to common naming conventions and settings

Reply:
Should I just removed the contribex dir and comment out the test for it? I just meant it as an example that this PR works :woman_shrugging: 

I agree that we shouldn't merge this as-is.

Reply:
Leaving the dir and everything is fine (and preferred!), if the approvers are changed to reviewers for now

Reply:
Done!